app/controllers/application_controller.rb

```diff
@@ -15,31 +15,37 @@ class ApplicationController < ActionController::Base
   rescue_from ActiveRecord::RecordNotFound, with: :not_found!
 
   def current_organization
-    return @current_organization if defined? @current_organization
+    return @current_organization if instance_variable_defined? :@current_organization
 
-    return @current_organization = nil unless current_role
-    return @current_organization = current_role.resource if current_role&.resource&.is_a?(Organization)
-
-    @current_organization = Organization.find_by(short_name: params[:organization_name])
+    @current_organization = if !current_role
+      nil
+    elsif current_role&.resource&.is_a?(Organization)
```

Comment:
FYI you never need & before `is_a?` because it's defined on everything, including nil. Nitpick because it was already like that.

Reply:
Yea actually, the line above already guards against `current_role` being `nil` so we could move both of these safe navigation operators.

Reply:
Fixed.